### DISCUSSION TITLE
Refresh object from database

### DISCUSSION THREAD

**Discussion by amolpatel** [prisma/prisma#12844] [Q&A] [answered]
Is there a straightforward way to refresh an object from the database without having to explicitly querying for it? I'm looking for something similar to [Django's `refresh_from_db` ](https://docs.djangoproject.com/en/4.0/ref/models/instances/#django.db.models.Model.refresh_from_db).

Example:
```javascript
const user = await prisma.user.findFirst({
   where: {
      id: userId,
   },
   include: {
      team: {
        include: { name: true },
      },
   },
});

await prisma.team.update({
  where: { id: teamId },
  data: {
     name: "New Team Name",
  }
});
```

Now, if I were to use `user.team.name` in my code without refetching the user, it would refer to the old value.

**Comment by austincrim** [comment, marked as answer]
Hey @amolpatel 👋 ,

We do not support a pattern like this currently. Could you open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) outlining more details?

Thanks!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
